src/app/pages/debrief/debrief.page.html

```diff
@@ -1,100 +1,94 @@
 <ion-header>
```

Comment:
Not sure why the formatting has changed so much, as this goes against the `indent_size = 2` on the `.editorconfig`